libs/libxml2/v2.10.3/wl-build.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+if [[ ! -v WASMLABS_ENV ]]
+then
+    echo "Wasmlabs environment is not set"
+    exit 1
+fi
+
+# export CFLAGS_CONFIG="-O3 -g"
+export CFLAGS_CONFIG="-O0"
+
+export CFLAGS_WASI="--sysroot=${WASI_SYSROOT} -I./wasmlabs-stubs -D_WASI_EMULATED_MMAN -D_WASI_EMULATED_GETPID -D_WASI_EMULATED_SIGNAL -D_WASI_EMULATED_PROCESS_CLOCKS"
+export LDFLAGS_WASI="--sysroot=${WASI_SYSROOT} -lwasi-emulated-mman -lwasi-emulated-getpid -lwasi-emulated-signal -lwasi-emulated-process-clocks"
+
+export CFLAGS_LIBXML2=''
+
+logStatus "Using LIBXML2 DEFINES: ${CFLAGS_LIBXML2}"
+
+export CFLAGS_BUILD='-D_POSIX_SOURCE=1 -D_GNU_SOURCE=1 -DHAVE_FORK=0 -DWASM_WASI'
```

Comment:
We can get rid of `WASM_WASI`, right?

Reply:
And here, again I would vote for trying to remove as many of these flags as possible.